psi4/src/psi4/libmoinfo/moinfo_base.cc

```diff
@@ -91,27 +90,23 @@ void MOInfoBase::compute_number_of_electrons() {
     nbel = nel - nael;
 }
 
-void MOInfoBase::compute_ioff() {
```

Comment:
Thanks for removing this. It's probably faster to compute `ioff` directly than to read it off from memory.